modules/interacting-serverless-apps-http2-gRPC.adoc

```diff
@@ -3,76 +3,55 @@
 // serverless/knative-serving/external-ingress-routing/using-http2-gRPC.adoc
 
 :_content-type: PROCEDURE
-[id="interacting-serverless-apps-http2-gRPC_{context}"]
+[id="interacting-serverless-apps-http2_{context}"]
```

Comment:
This isn't necessarily a merge blocker, but I think the previous ID was better because it matched the title and file name. If you're open to changing it back, please do. Thanks!